apps/playground-web/src/app/connect/pay/embed/LeftSection.tsx

```
@@ -13,17 +12,19 @@
 } from "lucide-react";
 import Link from "next/link";
 import type React from "react";
-import { useEffect, useState } from "react";
+import { useState } from "react";
+import { type Address, isAddress } from "thirdweb";
```

Biome (1.9.4)

[error] This import is unused.

Unused imports might be the result of an incomplete refactoring.
Safe fix: Remove the unused import.


(lint/correctness/noUnusedImports)